man/org.freedesktop.home1.xml

```diff
@@ -71,6 +71,9 @@ node /org/freedesktop/home1 {
       RegisterHome(in  s user_record);
       UnregisterHome(in  s user_name);
       CreateHome(in  s user_record);
+      CreateHomeEx(in  s user_record,
```

Comment:
doesn't matter, but so far we usually called extended functions like that FooWithBar(), where "bar" is the primary new feature we added. hence, for the sake of nitpicking, maybe call this CreateHomeWithBlobs?

but doesn't matter, i am fine either way

Reply:
hmm, grepping around i see that I am talking bs, there are plenty of bus calls called FoobarEx() already, so maybe just ignore me.